public/pull_request_template

```diff
@@ -0,0 +1,20 @@
+## Describe your changes
+
+## Issue ticket number and link
+
+## Name of Contributor(s)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist before requesting a review
```

Comment:
What do you all think? Checkboxes or maybe a process of how to contribute, or maybe just a link to an onboarding guide? :)